src/shared/components/Header/Navigation/styles.ts

```diff
@@ -36,9 +44,29 @@ export const SearchButton = styled.button`
   background-color: transparent;
   color: ${Colors.BLACK};
   font-weight: 500;
+  margin-right: 40px;
+  svg {
+    margin-right: 10px;
+  }
+  ${media.tabletMMax} {
+    order: 2;
+  }
+  ${media.mobileMax} {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 37px;
+    height: 37px;
+    font-size: 0;
```

Comment:
`font-size: 0;` looks a bit confusing to me? maybe to use some visibility: hidden?

Reply:
`visibility: hidden` this will completely hide the search button